src/app/zwiazek/historia-zwiazku/UnionAddendum.tsx

```diff
@@ -0,0 +1,16 @@
+interface AddendumSectionProps {
+  addendum: { paragraph: string | string[] };
+}
+
+export default function UnionAddendum({ addendum }: AddendumSectionProps) {
```

Comment:
Move those components to `app/../../_components`.

I think it looks ok, but also on the desktop it is quite small. Maybe we could enlarge it a little bit? Quite hard to read

Reply:
Also you can change it to arrow functions with implicit return. Those are not page functions. So no need for default exports as well (stick to named imports for everything that does not have to be default exported to suite NextJS requirements)

Reply:
Regarding `/_components` dir. Also thanks @NacomiTagiera for mentioning it during the meeting.
https://nextjs.org/docs/app/getting-started/project-structure#route-groups-and-private-folders